glide.lock

```diff
@@ -96,6 +99,8 @@ imports:
   - tnet
   - trand
   - typed
+- name: github.com/vektra/mockery
+  version: b6f6c13c25c3f0a17724553fa01580f328158d7a
```

Comment:
how would we prevent people to create their own mocks?

Reply:
Mock clients are generated and baked in to the dependency tree, you can create arbitrary mock clients by yourself, but the server in your integration test won't use those mocks.